### PR TITLE
154 update docs with 'getting started' 

### PR DESCRIPTION
**Pull request by spicy-sauce**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -2,77 +2,52 @@
...
-```bash
-wget https://datayoga.io/releases/dy_cli/latest/dy_cli-ubuntu18.04-latest.tar.gz -O /tmp/dy_cli.tar.gz
-```
-
-RHEL8
+## Create Python Virtual Environment
 
 ```bash
-wget https://datayoga.io/releases/dy_cli/latest/dy_cli-rhel8-latest.tar.gz -O /tmp/dy_cli.tar.gz
+python -m venv venv
```
Comment (zalmane) [inline docs/index.md]: change to 'python3'

File: docs/index.md [hunk 2]
```diff
@@ -2,77 +2,52 @@
...
-```
-
-UBUNTU18.04
-
-```bash
-wget https://datayoga.io/releases/dy_cli/latest/dy_cli-ubuntu18.04-latest.tar.gz -O /tmp/dy_cli.tar.gz
-```
-
-RHEL8
+## Create Python Virtual Environment
```
Comment (zalmane) [inline docs/index.md]: should we note this is optional?

File: datayoga-py/datayoga_core/resources/scaffold/jobs/sample/hello.yaml [hunk 3]
```diff
@@ -1,9 +1,19 @@
...
-      field: greeting
-      expression: "'hello ' || name"
-      language: sql
+      fields:
+        - field: full_name
+          language: jmespath
+          expression: concat([capitalize(fname), ' ' , capitalize(lname)])
+        - field: greeting
+          language: sql
+          expression: "'Hello ' || CASE WHEN gender = 'F' THEN 'Ms.' WHEN gender = 'M' THEN 'Mr.' ELSE 'N/A' END || ' ' || full_name"
```
Comment (zalmane) [inline datayoga-py/datayoga_core/resources/scaffold/jobs/sample/hello.yaml:14]: this is interesting. reusing a field within a field. this is what Informatica does. We should probably add a test for this.

File: docs/index.md (hunk 1 above)
Comment (spicy-sauce) [reply docs/index.md]: done

File: docs/index.md (hunk 2 above)
Comment (spicy-sauce) [reply docs/index.md]: not sure, what do you think? maybe we can remove it completely here and add such a section in the main README.

File: datayoga-py/datayoga_core/resources/scaffold/jobs/sample/hello.yaml (hunk 3 above)
Comment (spicy-sauce) [reply datayoga-py/datayoga_core/resources/scaffold/jobs/sample/hello.yaml:14]: why does it matter if this `full_name` field was defined here or in a previous block?

File: docs/index.md (hunk 2 above)
Comment (spicy-sauce) [reply docs/index.md]: removed

File: docs/index.md (hunk 1 above)
Comment (spicy-sauce) [reply docs/index.md]: actually removed this section, so irrelevant anymore.

**Pull request merged**